modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserAccountResourceTest.java

```diff
@@ -1941,6 +1953,39 @@ private void _testPatchUserAccountWithImageExternalReferenceCode()
 		Assert.assertTrue(patchUserAccount.getImageId() > 0);
 	}
 
+	private void _testPostAccountUserAccountsByExternalReferenceCodeByEmailAddressWithRoleId()
+		throws Exception {
+
+		List<User> users = Arrays.asList(
+			UserTestUtil.addUser(), UserTestUtil.addUser(),
+			UserTestUtil.addUser(), UserTestUtil.addUser());
+
+		AccountRole accountRole = _accountRoleLocalService.addAccountRole(
```

Comment:
move this after the first for loop (it's needed only for the "second part" of the test"